modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -18,8 +18,9 @@ any questions.
 	- [Toast](#toast)
 - [Utility Updates](#utility-updates)
   - [focusRing](#focusring)
-- [Token Updates](#token-updates)
-	- [Depth](#depth)
+	- [useThemedRing](#useThemedRing)
+	- [useThemeRTL](#useThemeRTL)
+- [Token Updates](#token-updates) - [Depth](#depth)
```

Comment:
This may need to be addressed. James helped me out with this since it's not a Tab it's just spaces that don't allow it to format itself

Reply:
dang it

Reply:
good catch, fixed!